cfe_internal/enterprise/CFE_knowledge.cf

```diff
@@ -160,12 +160,27 @@ bundle agent cfe_internal_setup_knowledge
       create => "true",
       perms => mog("0640", $(def.cf_apache_user), $(def.cf_apache_group));
 
+      # Current MP Default: -rw-r--r--. 1 cfapache cfapache 351 Nov 11 17:24 log-2021-11-11.log
+      "$(sys.workdir)/httpd/logs/application/." -> { "ENT-7730" }
+        comment => "Ensure permissions for $(sys.workdir)/httpd/logs/application/.*",
+        handle => "cfe_internal_setup_knowledge_files_httpd_application_log_files",
+        file_select => plain,
+        depth_search => recurse( "inf" ),
+        perms => mog("0600", $(def.cf_apache_user), $(def.cf_apache_group));
+
       "$(sys.workdir)/httpd/logs/."
       comment => "Ensure permissions for $(sys.workdir)/httpd/logs",
-      handle => "cfe_internal_setup_knowledge_files_httpd_logs",
+      handle => "cfe_internal_setup_knowledge_files_httpd_logs_dir",
       create => "true",
       perms => mog("0640", $(def.cf_apache_user), $(def.cf_apache_group));
 
+      "$(sys.workdir)/httpd/logs/." -> { "ENT-7730" }
+        comment => "Ensure permissions for $(sys.workdir)/httpd/logs",
+        handle => "cfe_internal_setup_knowledge_files_httpd_log_files",
+        file_select => plain,
+        depth_search => recurse_with_base( "0" ),
+        perms => mog("0600", root, root);
```

Comment:
So the logging part of Apache runs as *root*?

Reply:
yeh, seems so